File: 小作品/Game/design/src/main/java/com/xfhy/design/MainActivity.java
package com.xfhy.design;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

/**
* Design
* @author xfhy
* create at 2017年6月23日16:47:39
*/
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void jumpScrolling(View view) {
        Intent intent = new Intent(this, ScrollingActivity.class);
        startActivity(intent);
    }
}

File: KotlinTest/app/src/main/java/com/xfhy/kotlintest/MainActivity.kt
package com.xfhy.kotlintest

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

/**
 * @author xfhy
 *
 */
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val focecastList = findViewById(R.id.forecast_list) as RecyclerView

    }

}
